backend/docs/performance/future_optimizations.md

```
@@ -0,0 +1,230 @@
+
+### 採用条件
+以下の条件を満たす場合のみ検討：
+
+- **B1+B2実装後も5秒以上かかる**
+- ユーザー数が増加してストレージコストが許容範囲内（月額$10未満）
+- Ingest時間の増加が許容できる（現在5分 → 6分）
+
+### 実装の段階
+
+#### MVP: 月次トップ100のみ
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)